sdcm/tester.py

```diff
@@ -1620,7 +1627,7 @@ def run_stress_cassandra_thread(
                                           profile=profile,
                                           node_list=self.db_cluster.nodes,
                                           round_robin=round_robin,
-                                          client_encrypt=self.db_cluster.nodes[0].is_client_encrypt,
```

Comment:
why not just use `self.params.get('client_encrypt')`, the rest of SCT code works according to that and not by `is_client_encrypt`, with one exception in the manager create cluster code.

I don't see the reason not to follow configuration here.

Reply:
I wasn't sure either, but I assumed there must be a reason as to why. Are you certain there's no need for this check?

Reply:
this function is being used twice in the code, here and in the manager code.
in the rest of the places in the code (other stress commands, cqlsh, driver initialization) we use the parameter, and not checking what's the configuration of the node. (i.e. we configure the node base on the parameter)

Reply:
I switched to client_encrypt